ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -62,12 +62,23 @@ if [[ "${CLUSTER_TYPE}" == gcp ]]; then
     popd
 fi
 
-test_suite=openshift/conformance/parallel
-if [[ -e "${SHARED_DIR}/test-suite.txt" ]]; then
-    test_suite=$(<"${SHARED_DIR}/test-suite.txt")
-fi
-
-openshift-tests run "${test_suite}" \
-    --provider "${TEST_PROVIDER}" \
+if [[ "${OPENSHIFT_UPGRADE:-false}" == "false" ]]; then
```

Comment:
Hrm.  Not clear to me how much we want to pack into `OPENSHIFT_UPGRADE` here with an opinionated conditional inside the step.  How about #9676 and then setting:

```sh
OPENSHIFT_UPGRADE=true  # for the install switch.
TEST_COMMAND=run-upgrade
TEST_SUITE=all
```

Or maybe even RELEASE_IMAGE=RELEASE_IMAGE_INITIAL and then use indirect expansion:

```bash
$ RELEASE_IMAGE_INITIAL=A
$ RELEASE_IMAGE_LATEST=B
$ RELEASE_IMAGE=RELEASE_IMAGE_INITIAL
$ echo "${!RELEASE_IMAGE}"
A
```

And then setting those variables appropriately when we define the job?